### PR TITLE
chore(devnet): use westend-local for docker local devnets

### PR DESCRIPTION
**Pull request by jimjbrettj**
## Changes

- Use westend local for devnet
- Upgraded substrate dockerfile nodes to use westend v0.9.37
- Note: it seems both here and on dev that for the cross client devnet we have a below threshold issue

## Tests

```sh
Test both gossamer and cross client devnets to make sure behavior seems normal
```

## Primary Reviewer

@timwu20

### CODE REVIEW

**Review by qdm12 — APPROVED** [review]

**Review by EclesioMeloJunior — COMMENTED** [review]
Nice changes, I made some tests and everything was fine! 🚀 
image

File: devnet/substrate_bob.Dockerfile [hunk 1]
```diff
@@ -12,13 +13,10 @@ COPY ./devnet/go.mod ./devnet/go.sum ./
 RUN go mod download
 
 COPY ./devnet .
-RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alice > conf.yaml
+#RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alice > conf.yaml
```
Comment (EclesioMeloJunior) [inline devnet/substrate_bob.Dockerfile:16]:
```suggestion
RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alice > conf.yaml
```

File: devnet/substrate_bob.Dockerfile [hunk 2]
```diff
@@ -42,11 +40,11 @@ COPY --from=openmetrics /devnet/conf.yaml /etc/datadog-agent/conf.d/openmetrics.
 
 USER polkadot
 
-COPY ./devnet/chain ./chain/
+COPY ./chain/ ./chain/
 
-ENTRYPOINT service datadog-agent start && /usr/bin/polkadot \
+ENTRYPOINT  /usr/bin/polkadot \
```
Comment (EclesioMeloJunior) [inline devnet/substrate_bob.Dockerfile:45]:
```suggestion
ENTRYPOINT  service datadog-agent start && /usr/bin/polkadot \
```

**Review by EclesioMeloJunior — CHANGES_REQUESTED** [review]

File: devnet/substrate_alice.Dockerfile [hunk 3]
```diff
@@ -16,9 +16,6 @@ RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alic
 
 FROM parity/polkadot:${POLKADOT_VERSION}
 
-ARG POLKADOT_VERSION
-# Using a genesis file with 3 authority nodes (alice, bob, charlie) generated using polkadot $POLKADOT_VERSION
-ARG CHAIN=3-auth-node-${POLKADOT_VERSION}
```
Comment (EclesioMeloJunior) [inline devnet/substrate_alice.Dockerfile:21]: Same here, could we keep the `ARG CHAIN=westend-local` and use it in the `ENTRYPOINT` command?

**Review by EclesioMeloJunior — CHANGES_REQUESTED** [review]

**Review by EclesioMeloJunior — APPROVED** [review]

**Pull request merged**